update_existing_families.py

```diff
@@ -149,8 +160,68 @@ def load_csv_file(filename, expected_headers):
                         'OTHER_DATE': ''
                     }
                     dp_import_studentrecords.append(studentrecord)
+
     if len(donor_ids_encountered) == 0:
-        print("TODO Create record for new donor / new student %s and add to dp_import_studentrecords" % (student_id))
+        # At this point we know that: 
+        #   (a) this student isn't in DP (i.e., this is new student)
+        #   (b) this student doesn't have an sibling currently in BSD
+        # Chances are the donor is also new, unless 
+        #   (i) the donor has a kid that has graduated already OR, 
+        #   (ii) this student is returning after a break from BSD, but somehow was given a new student ID
+        # If either (i) or (ii) is true, then DP will not create a new record, but update the existing donor record, so we should be okay. 
+        # At any rate, we have to prepare a new record for this donor, with several custom fields
+        print("Creating record for new donor w/ new student %s" % (student_id))
```

Comment:
I was thinking exactly the same thing. We should only emit some succinct stats to stdout